pgvectorscale/src/access_method/mod.rs

```diff
@@ -62,7 +61,11 @@ fn amhandler(_fcinfo: pg_sys::FunctionCallInfo) -> PgBox<pg_sys::IndexAmRoutine>
     amroutine.ampredlocks = false;
     amroutine.amcanparallel = false; //TODO
     amroutine.amcaninclude = false; //TODO
-    amroutine.amusemaintenanceworkmem = false; /* not used during VACUUM */
+    #[cfg(any(feature = "pg13", feature = "pg14", feature = "pg15", feature = "pg16"))]
+    {
+        amroutine.amoptsprocnum = 0;
+        amroutine.amusemaintenanceworkmem = false; /* not used during VACUUM */
+    }
```

Comment:
You don't need this change anymore (or any of the others in this file), since this is only relevant for supporting PG 12

Reply:
fixed